recipes/ffn/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "ffn" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4a79e72e06ff328e333ffe97010b1ce110bcd694fcd03be7351bf5065cd273e8 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - decorator >=4
+    - future >=0.15
+    - matplotlib >=1
+    - numpy >=1.5
+    - pandas >=0.19
+    - pandas-datareader >=0.2
+    - scikit-learn >=0.15
+    - scipy >=0.15
+    - tabulate >=0.7.5
+    - python >=3.6
+
+test:
```

Add a check to verify that the run dependencies match the Python ones.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```